### ISSUE TITLE
Mobile Menu Voice Over

### ISSUE THREAD

**Issue by mumptastic** [TAMULib/tamu-library-components#428]
Trying to use the mobile menu of the library site are not accessible for users using the 'Voice Over' features in iOS and Android. 
Visually impaired individuals never get the prompt to tap on the menu. The button is ignored completely.
